scripts/fixtures.sql

```diff
@@ -21,8 +30,78 @@ begin
     end loop;
 end; $$;
 
-call department('Mathematics');
-call department('Computer Science');
-call department('Physics');
+create procedure create_departments()
+language plpgsql
+as $$
+declare
+    root realms.id%type;
+begin
+    insert into realms (name, parent, path_segment)
+        values ('Lectures', 0, 'lectures')
```

Comment:
`insert`-s are now formatted inconsistently. I tend to not indent the `values` clause, but also don't really have a hard opinion either way. I have **seen**

```sql
insert into ...
values ...
```

much more often than

```sql
insert into ...
    values ...
```

though.

Reply:
I went with the indented version as I find it a lot easier to read. And regarding what is used more in the world: I WONDER IF THERE IS ANOTHER THING MOST SQL CODE DOES THAT WE DO NOT AGREE WITH...